packages/turf-isobands/index.d.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="geojson" />
+
+type Points = GeoJSON.FeatureCollection<GeoJSON.Point>;
+type LineStrings = GeoJSON.FeatureCollection<GeoJSON.LineString>;
+
+/**
+ * http://turfjs.org/docs/#isolines
+ */
+declare function isobands(points: Points, z: string, resolution: number, breaks: Array<number>): LineStrings;
+declare namespace isolines { }
```

Comment:
namespace must be the same name as the function name.

This breaks the typescript definition.